<!doctype html>

<html lang="en">
<html lang="en">
<head>
<!-- Google tag (gtag.js) -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-MPQZSVEQWQ"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-MPQZSVEQWQ');
</script>
	<!--KaTeX-->
	<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/katex.min.css" integrity="sha384-AfEj0r4/OFrOo5t7NnNe46zW/tFgW6x/bCJG8FqQCEo3+Aro6EYUG4+cU+KJWu/X" crossorigin="anonymous">
	<script defer src="https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/katex.min.js" integrity="sha384-g7c+Jr9ZivxKLnZTDUhnkOnsh30B4H0rpLUpJ4jAIKs4fnJI+sEnkvrMWph2EDg4" crossorigin="anonymous"></script>
	<script defer src="https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/contrib/auto-render.min.js" integrity="sha384-mll67QQFJfxn0IYznZYonOWZ644AWYC+Pt2cHqMaRhXVrursRwvLnLaebdGIlYNa" crossorigin="anonymous"></script>
	<script>
	    document.addEventListener("DOMContentLoaded", function() {
	  	  renderMathInElement(document.body, {
	  	  });
	    });
	</script>
	<meta charset="utf-8">
	<meta name="viewport" content="width=device-width, initial-scale=1">

	<title>Melding Data with Science: A Perspective on the Recent Nobel Prizes</title>
	<meta name="description" content="The advent of artificial intelligence (AI) is altering paradigms faster than ever. In hindsight, the 2024 Nobel Prizes in Physics and Chemistry were a recognition of this remarkable phase of human endeavour. This essay attempts to acquaint the general reader to underpinning developments that has led to such a surge. Creative harnesses that facilitated the Nobel recognitions are invoked, along with emergent opportunities and subtle caveats.">

	<meta property="og:title" content="Melding Data with Science: A Perspective on the Recent Nobel Prizes">
	<meta property="og:type" content="website">
	<meta property="og:url" content=http://localhost:4000/issue2/nobel/>
	<meta property="og:description" content="The advent of artificial intelligence (AI) is altering paradigms faster than ever. In hindsight, the 2024 Nobel Prizes in Physics and Chemistry were a recognition of this remarkable phase of human endeavour. This essay attempts to acquaint the general reader to underpinning developments that has led to such a surge. Creative harnesses that facilitated the Nobel recognitions are invoked, along with emergent opportunities and subtle caveats.">
	<meta property="og:image" content="">

	<link rel="shortcut icon" type="image/x-icon" href="/favicon.ico?">

	<!doctype html>

<link rel="stylesheet" href="/assets/css/styles.css">
<link rel="stylesheet" href="/assets/webfonts/Lato/stylesheet.css">
<link rel="stylesheet" href="/assets/webfonts/Hero New/stylesheet.css">
<link rel="stylesheet" href="/assets/webfonts/Roboto/stylesheet.css">
<link rel="stylesheet" href="/assets/webfonts/SymbolsNF/stylesheet.css">
<link rel="stylesheet" href="/assets/webfonts/Lilex/stylesheet.css">

</head>


<body>
<header>
<nav>
<a id="home-button" href="/">
 <img src="/assets/global/bannerTextWhite.svg"/>
</a>
<a href="/magazine/">Magazine</a>
<a href="/archives/">Archives</a>
<a href="/games/">Games</a>
<a href="/submit/">Submit</a>
<a href="/about/">About</a>
</nav>
</header> 


<main>
  
  <div class="hero-image-caption">
	  <img class="hero-image" src="/assets/images//issue2/nobel//nobel2024.png"/>
  
  </div>

<h1> Melding Data with Science: A Perspective on the Recent Nobel Prizes </h1>

<div class="author">Prof. Neelanjana Sengupta</div>
<div class="affiliation">IISER Kolkata</div>
<div class="date">Published: Feb 02, 2025</div>


<div class="excerpt">
<p>The advent of artificial intelligence (AI) is altering paradigms faster than ever. In hindsight, the 2024 Nobel Prizes in Physics and Chemistry were a recognition of this remarkable phase of human endeavour. This essay attempts to acquaint the general reader to underpinning developments that has led to such a surge. Creative harnesses that facilitated the Nobel recognitions are invoked, along with emergent opportunities and subtle caveats.</p>

</div>

<hr>
<p>Prior to his death, Alfred Nobel, the Swedish scientist, inventor, and entrepreneur, instituted his namesake awards for those who ‚Äúshall have conferred the greatest benefit to humankind.‚Äù Worldwide, the Nobel prizes count amongst the highest achievable recognitions; they elevate individuals for life and bring immense prestige to associated institutions (1). Historically, the Prizes in the natural sciences have been associated with ‚Äòfundamental discoveries‚Äô in sub-disciplines within Physics, Chemistry, and Physiology (or Medicine), typically arriving decades after the initial revelation. Speculative conjectures about the nature of the prizes run rife until their announcements in October each year.</p>

<p>By standard measure, the 2024 Nobel Prizes to John Hopfield and Geoffrey Hinton in Physics (‚Äúfor foundational discoveries and inventions that enable machine learning with artificial neural networks‚Äù) to David Baker (‚Äúfor computational protein design‚Äù), and to Demis Hassabis and John Jumper in Chemistry  (‚Äúfor protein structure prediction‚Äù), are unconventional. Not only do the citations defy disciplinary expectations, but part of the Chemistry award is for research reported about half a decade ago (2). Strikingly, John Jumper (born 1985) is the youngest Chemistry laureate in the last seven decades. Collectively, the prizes are an unequivocal nod to artificial intelligence (AI), a product of the human mind arguably poised to overtake  human creativity (3).</p>

<p>This essay attempts to thread the inception and evolution of AI and Artificial Neural Networks (ANN) in the context of the two 2024 Prizes and their culmination to sine qua non in science and life. While the writing does not engage in pedagogy, the references should adequately orient the reader for deeper appreciation.</p>

<h2 id="inception">Inception</h2>
<p>Artificial Intelligence (AI) represents a growing spectrum of data-based algorithmic technologies that aim to replicate decision-making and other nuances of the human mind. Mathematical recapitulation of the brain‚Äôs ‚Äúnervous activity‚Äù is originally attributed to the fortuitous crossing of paths between a neuropsychologist, Warren McCulloch, and a self-taught logician, Walter Pitts. As early as 1943, McCulloch and Pitts suggested that the brain‚Äôs neural events manifest as calculable logic (4). Their analogy, schematically described in Fig 1, between synaptic activity and logic gates led to the concept of ‚Äúneural nets‚Äù and marked the epoch of artificial neural networks (ANN). Close on the heels of McCulloch-Pitts, in 1949, psychologist Donald Hebb published ‚ÄúThe Organization of Behaviour.‚Äù Hebb‚Äôs prescient text related the influence of the environment on behaviour and related synaptic organization to network dynamics (5, 6). Hebb evinced that learning and memories relate to the co-activation of multiple neurons.</p>

<figure>
  
  
  
  
  <img src="/assets/images/issue2/nobel/neuron.png" width="400" />
  
  
  
  
  <figcaption> FIG 1.  Schematic depiction of McCulloch-Pitts Model, (adapted from (47)).  The inputs (violet) bring in signals from neighbouring neurons, acting as the dendrites. The summing function mimics the cell body by generating impulses that are passed along the axon. The connection from the output to other artificial neurons represents the axon.
</figcaption>
  
</figure>

<p>The combined insights of McCulloch, Pitts, and Hebb were foundational to the development of AI. However, the next three decades observed ebbs and rises due to disinterested funding agencies. Luckily, more exciting developments in computational hardware helped sustain interest in ANNs. Notable work from this era includes Frank Rosenblatt‚Äôs implementable feedforward network for image interpretation (7). His introduction of the ‚Äúperceptron‚Äù was a direct attempt to mimic the brain‚Äôs neural structures. While attractive in its simplicity, the original perceptron died a natural death owing to limitations enunciated by Marvin Minsky and Seymour Papert (8). The intermingling of various disciplines during these early endeavours is evident from the article curiously titled  ‚ÄúWhat the Frog‚Äôs Eye Tells the Frog‚Äôs Brain,‚Äù published in 1959 in the Proceedings of the Institute of Radio Engineers (9).</p>

<h2 id="the-hopfield-network-and-the-boltzmann-machine">The Hopfield Network and the Boltzmann Machine</h2>
<p>John Hopfield began his career in solid-state physics. A science maverick, he meandered from physics to biochemistry and spectroscopy prior to attempts at replicating the brain‚Äôs workings. It is interesting that at the time of the 2024 Prize, he was (and still is) the Howard A. Prior Professor of Molecular Biology (Emeritus) at Princeton University. It is unsurprising to attribute the remark, ‚ÄúI didn‚Äôt really think of this as moving into biology, but rather as exploring another venue in which to do physics,‚Äù to this polymath (10).</p>

<p>Hopfield was inspired by collective phenomena in physics commonly studied by the lattice-based Ising model. Noting that an array of magnetic spins respond collectively when exposed to a perturbation (such as a magnetic field), Hopfield reasoned that interconnected neurons would elicit similar responses when subjected to an impulse (such as an image or sound). The basic Hopfield network was thus born (11). The model assigned input layers (akin to dendrites), weights (akin to synapses), and nodes (akin to the neuronal centre) to an elementary ANN. The nodes were assigned a binary number (0 or 1), and their energy was deduced from a linear combination of the weights and values of other interconnected nodes; see Fig 2. The result was a dynamical recurrent network for associative memory. Hopfield then advanced his model for graded response incorporating continuous-time dynamics (12). Over the next few years, Hopfield consolidated and refined his model in association with other neuroscientists such as David Tank.</p>

<figure>
  
  
  
  
  <img src="/assets/images/issue2/nobel/hopfieldNetwork.png" width="300" />
  
  
  
  
  <figcaption> FIG 2.  A Hopfield network consists of a single layer of neurons (represented by black and gray nodes), with connectivities (red lines). The neurons adopt only two values. Adapted from (48).
</figcaption>
  
</figure>

<p>The early Hopfield model settled into energy valleys (or minima); this was partially addressed by the introduction of a fictive ‚Äòtemperature,‚Äô T. Leveraging the idea of thermal stochasticity, another physicist, Geoffrey Hinton, introduced the concept of the Boltzmann machine (Fig 3), replacing Hopfield‚Äôs binary nodes with weights (13). Along with local biases, the modified network now represented statistical distributions of patterns, with an energy E and T-dependent Boltzmann-like probabilities of states. Large numbers of hidden units in the Boltzmann machine learned complex associations. Remarkably, this development marked neural network shifts to generative models capable of generating new data based on the training patterns. Hinton developed the idea of backpropagation, whereby hidden units not connected to the input or output help refine the network (14).</p>

<h2 id="deeper-and-denser">Deeper and Denser</h2>
<p>The original breakthroughs saw successful applications in image recognition, language modelling, and clinical data. Important work in this era by LeCun, Benjio, and Fukushima (15, 16, 17) led to the development of deep convolutional networks. However, training efficient yet deep multilayered networks remained a recurring challenge. The next breakthrough, vis-a-vis Hinton again, arrived in the form of the Restricted Boltzmann Machine (RBM). The RBM (Fig 4) eliminated connectivities (weights) between nodes in the same layer and led to the emergence of the much faster contrastive divergence method (18). This marked the advent of deep neural networks with dense interconnectivity between millions of neurons that continue to develop for performing previously unimaginable tasks (19).</p>

<figure>
  
  
  
  
  <img src="/assets/images/issue2/nobel/boltzmannMachine.png" width="400" />
  
  
  
  
  <figcaption> FIG 3.  Schematic of a Boltzmann machine as a grid of interconnected nodes, with the clear nodes representing visible nodes and the blurred ones representing hidden nodes. The visible nodes are linked to the hidden nodes (illustrated by additional blue lines between the layers), indicating their influence on the network‚Äôs function.  Adapted from (48).
</figcaption>
  
</figure>

<p>Today, AI methods with underlying DNNs are indispensable in large segments of human endeavour. Within the parent category of one of the 2024 prizes, AI has aided discoveries in plasma physics and biological physics (20). More easily recognizable applications today include image and voice recognition, language processing and translation, market recommendations, customer-relationship management, health data management, and human health and drug discovery (21). The last aspect bears upon the other focal point of this essay, namely the 2024 Prize in Chemistry.</p>

<h2 id="folding-the-machineries-of-life">Folding the Machineries of Life</h2>
<p>Life is underwritten by molecules obeying physical laws. Some of these molecules are information templates, while others synchronize and execute the underwritten rules. Key participants in this complex machinery are proteins, chains of twenty naturally occurring units (amino acids) that perform myriad duties that range from facilitating chemical reactions to acting as scaffolds and signal transmitters to offering defences and preventing organismal damage. To carry out their mandate, the vast majority of these voracious workers must acquire very specific three-dimensional shapes or structures within a very short time. In the mid-twentieth century, Christian Anfinsen demonstrated, with a series of innovative experiments, how the acquisition of specific structures was a direct result of the sequence of the amino acids (22). Anfinsen was awarded the Chemistry Nobel Prize in 1972.</p>

<p>Over seven odd decades, the protein folding problem has intrigued, attracted, and flummoxed communities of science practitioners (23). Cyrus Levinthal‚Äôs thought experiment, ca. 1968, highlighted how, if random sampling of all possible conformations were allowed, the folding time would likely exceed the age of the universe. Levinthal‚Äôs paradox inspired crucial physical explanations (24) alongside vast experimental efforts (25). Yet, a solution to the protein folding problem remained elusive, limiting advancement not just in medicine and therapeutics but in fundamental progress as well.</p>

<p>Though aided in part by the advent of computational techniques, capturing the precisely folded protein state has predominantly been deemed an experimental prerogative. Traditionally, the gold standard was X-ray crystallography, which reported the relative positions (coordinates) of individual protein atoms. Additionally, complementary techniques such as nuclear magnetic resonance (NMR) and low-temperature cryo-electron microscopy (cryo-EM) have offered crucial biological insight (26). Collectively, experimental advances underlie the rapid rise in the number of reported entries in the protein structural database (PDB), reaching 0.23 million entries in about three decades (27). It is worth noting that this milestone was achieved with tremendous investment in laboratory resources and human effort. It was realized that sole reliance on experimental methods would not break barriers or induce paradigm shifts in protein structural elucidation.</p>

<figure>
  
  
  
  
  <img src="/assets/images/issue2/nobel/restricted.png" width="400" />
  
  
  
  
  <figcaption> FIG 4.  Schematic of a Restricted Boltzmann Machine is similar to a Boltzmann machine but lacks connections within the same layer. RBMs are foundational in deep learning, often used as building blocks for deep belief networks.  Adapted from (48).
</figcaption>
  
</figure>

<h2 id="alphafold-can-we-replace-the-lab">AlphaFold: Can we Replace the Lab?</h2>
<p>Founded as a start-up in 2010, DeepMind aimed to transform the field of AI. Led by Demis Hassabis, it achieved early success in Deep Reinforced Learning. Taken over by the technology giant Google a few years later, DeepMind‚Äôs modules were defeating world champions in games such as Go, a Japanese game akin to Chess. Powered by other successes such as the AlphaStar and the Wavenet, DeepMind segued to protein structure prediction in 2020.</p>

<p>Freshly armed with a PhD in theoretical chemistry from the University of Chicago, John Jumper joined DeepMind in 2017. Prior to graduate school, Jumper had spent a few years at D.E. Shaw Research, a computational drug development company based in New York. At DeepMind, Hassabis and Jumper assembled a diversely trained team of mathematicians, physicists, chemists, and computer scientists. This team would go on to create an AI tool that could predict protein structure with only the amino acid sequence as input (29). At the Critical Assessment of Protein Structure (or CASP) biennial competition in 2018, this tool was able to best the next entry (from academia) by a long measure. Emboldened, the team then redesigned their neural network architecture to emerge with their now famous tool, AlphaFold. The new AI technology not only topped CASP in 2020 but yielded structures virtually identical to those reported via actual experiments. The algorithm, published in 2021 (30), unveiled the era of AI as a potential replacement for human effort in an actual wet lab (31). Later versions of AlphaFold have been empowered for the deduction of protein structures complexed with other molecules of biological significance (32), hugely impacting fields such as drug discovery and therapeutics. Despite some reasonable scepticism (see concluding section), AlphaFold‚Äôs value stands ratified not just by the Nobel recognition but also by a citation surge at nascency.</p>

<p>Large segments of the human proteome had awaited inspiration, resources, and experimental structural resolution. Soon after the publication of their algorithm, the AlphaFold team quickly moved to deduce a previously unimagined number (exceeding 200 million) of structures based on their known sequence. As of today, this technology has helped fill vast structural gaps within extant genomes, giving rise to an ‚ÄòAlphaFold database‚Äô (33). Such a leap in structural information is emerging as an important crucible for protein-based therapeutic discovery (34, 35)</p>

<h2 id="extra-genomic-design">Extra-Genomic Design</h2>
<p>Protein function is, in large part, enabled by the intricate details of structure. The backbone fold alignment and orientations of the sidechain cooperate to execute precise roles, such as priming molecules and ligands for chemical reactions. De novo protein design is, in some sense, the reverse of the folding problem - the function informs the fold, which in turn dictates the sequence. Importantly, this approach is not limited to sequences found in nature and does not demur from extra-genomic design.</p>

<p>The first artificial sequences were inspired by folds prevalent in natural proteins (36, 37). Early efforts culminated in the design of the first artificial protein, ‚ÄòTop7‚Äô, whose predicted structure was subsequently validated by experiment (38). The main architect, David Baker, owed this success to his own computational program, Rosetta, released a few years earlier in 1999 (39). While this program did not leverage any machine learning methods, it synchronized available structural data with physics-based methods. In the years following, Baker assembled large teams to progressively predict, design, and validate ranked structural folds capable of executing functions outside those dictated by nature. Unlike the AlphaFold team, the team includes scientists trained in experimental techniques. Today, Baker‚Äôs Institute for Protein Design iterates between targeting new functions to AI-based algorithm development to design proteins that are not the natural products of evolution (40). Emergent developments through this pipeline include anti-toxins (such as those against snake venom), synthetic receptors, protein-based nanomaterials, and customized vaccines. Baker is also credited with popularizing the protein folding problem via the crowdsourced online tool Foldit (41). Based on the original Rosetta, this publicly available computer game offers one the chance to leverage their ingenuity and create unique protein configuration structures with novel functions (42).</p>

<figure>
  
  
  
  
  <img src="/assets/images/issue2/nobel/alphafold.png" width="600" />
  
  
  
  
  <figcaption> FIG 5.  Protein structures predicted by the AlphaFold2 algorithm are presented for a diverse set of proteins. These include a chaperone, HSP14 (a); an enzyme, Abselson tyrosine kinase, or Abl (b); a transmembrane potassium channel, KcSA (c);a transmembrane copper transporter, ATP7B (d); and an intrinsically disordered protein, DISC1 (e). Experimental structures, where available in the protein data bank database, are superimposed in grey. Backbone root mean squared deviations (RMSD, in √Ö units) are provided. For the transporters, the surrounding cellular membrane bilayer is putatively modeled, and depicted in yellow (lipid headgroups) and pink (aliphatic tails). The inability of AlphaFold2 to predict alternate (polymorphic) structures is demonstrated by the omission of the segment corresponding to the inactive state of Abl (see b; in blue).
</figcaption>
  
</figure>

<h2 id="into-the-future">Into the Future</h2>
<p>In the last decade, artificial intelligence-based technologies have permeated human lives at previously inconceivable levels. The average individual, rural or urban, is exposed to, and at the same time, unknowingly contributes to, ever-expanding data that inform further development. Everyday interfaces include search engines, voice assistants, social media, and guided choices in digital shopping. Somewhat less routinely, people may use AI-assisted genealogy services or rely on AI-assisted diagnostics for acute diseases. Generative AI methods, such as ChatGPT that replicate human responses have generated further excitement.</p>

<p>While the horizons expand, it may be prudent to tread with caution, at least for the time being. While AI aids both knowledge acquisition and application, it appears that the effects may culminate non-uniformly (43). Effects on early exposure and over-usage may modulate the natural receptivity and intelligence of young brains; potential evolutionary consequences may be indeterminable. Blurred lines between reality and artificially generated scenarios may affect finely balanced societal, political, and legal environments. For those in proteomic work, it is important to keep in mind the limitations of tools such as AlphaFold. While quick and efficient and undoubtedly a boon for structural biology and structure-based drug design, AlphaFold is primarily a prediction algorithm that does not inform about the physico-chemical drivers of protein folding (43, 44, 45). Emergent work describes the limitations of this tool for effectively predicting protein folds associated with chemical staples or those with high disorder (see Fig 5).</p>

<p>Despite purported warnings, the era of artificial intelligence has arrived to thrive. One hopes this era will manifest as an ode to those ‚Äúwho confer the greatest benefit‚Äù to humanity.‚Äù</p>

<h2 id="postscript">Postscript.</h2>
<p>As of this writing in late January 2025, the large language model-based AI tool, DeepSeek, was sending ripples with its seemingly insurmountable capabilities. Operable at far lower computational cost, DeepSeek significantly supersedes its closest competitors (46). DeepSeek‚Äôs unveiling led to major market upheavals and catapulted AI into yet another paradigm shift.</p>



<hr>
  <div class="author-bio">
	
	
	
    <p><strong>Prof. Sengupta</strong> is a strong proponent of interdisciplinary and collaborative research. Her group at IISER Kolkata (mCED) probes molecular machinations of diverse biological systems with physics and data based methods.</p>

  </div>

<br>

  <div class="author-bio"><p>Discussions with mCED research group members, and critical feedback from Dr. Kumar Vanka (CSIR-NCL, Pune) is acknowledged.¬†Abhirup¬†Mukherjee and Pousukhi Bagchi are credited with Figs 1-4, and with Fig 5, respectively.</p>
</div>

<br>


<h3>References</h3>
<div class="references">
<ol>


<li id="ref-1">M. von Zedtwitz, T. Gutmann, and P. Engelmann, Research Policy, vol. 54, no. 1, p. 105150, Nov. 2024. </li>


<li id="ref-2">J. Jumper et al., Nature, vol. 596, no. 7873, pp. 583‚Äì589, Jul. 2021</li>


<li id="ref-3">W. D. Heaven, ‚ÄúGeoffrey Hinton tells us why he‚Äôs now scared of the tech he helped build,‚Äù MIT Technology Review, May 02, 2023</li>


<li id="ref-4">W. S. McCulloch and W. Pitts, ‚ÄúA logical calculus of the ideas immanent in nervous activity,‚Äù The Bulletin of Mathematical Biophysics, vol. 5, no. 4, pp. 115‚Äì133, Dec. 1943</li>


<li id="ref-5">D.O. Hebb, The organization of behavior (Wiley & Sons, 1949)</li>


<li id="ref-6">D. O. Hebb, ‚ÄúThe Organization of Behavior,‚Äù Apr. 2005</li>


<li id="ref-7">F. Rosenblatt, Principles of neurodynamics: Perceptrons and theory of brain mechanisms (Spartan Book, Washington D.C., 1962)</li>


<li id="ref-8">M.L. Minsky and S.A. Papert, Perceptrons: An introduction to computational geometry (MIT Press, Cambridge, 1969)</li>


<li id="ref-9">J. Lettvin, H. Maturana, W. McCulloch, and W. Pitts, ‚ÄúWhat the Frog‚Äôs Eye Tells the Frog‚Äôs Brain,‚Äù Proceedings of the IRE, vol. 47, no. 11, pp. 1940‚Äì1951, Nov. 1959</li>


<li id="ref-10">J. J. Hopfield, ‚ÄúTwo cultures? Experiences at the physics-biology interface,‚Äù Physical Biology, vol. 11, no. 5, pp. 053002‚Äì053002, Oct. 2014</li>


<li id="ref-11">J. J. Hopfield, ‚ÄúNeural networks and physical systems with emergent collective computational abilities.,‚Äù Proceedings of the National Academy of Sciences, vol. 79, no. 8, pp. 2554‚Äì2558, Apr. 1982</li>


<li id="ref-12">J. J. Hopfield, ‚ÄúNeurons with graded response have collective computational properties like those of two-state neurons,‚Äù Proceedings of the National Academy of Sciences, vol. 81, no. 10, pp. 3088‚Äì3092, May 1984</li>


<li id="ref-13">S.E. Fahlman, G.E. Hinton and T.J. Sejnowski. In Proceedings of the AAAI-83 conference, pp. 109-113 (1983)</li>


<li id="ref-14">D. E. Rumelhart, G. E. Hinton, and R. J. Williams, ‚ÄúLearning representations by back-propagating errors,‚Äù Nature, vol. 323, no. 6088, pp. 533‚Äì536, Oct. 1986</li>


<li id="ref-15">Y. LeCun, B. Boer, J.S. Denker, D. Henderson, R.E. Howard, W. Hubbard and L.D. Jackel, Neural Comput. 1, 541 (1989)</li>


<li id="ref-16">Y. LeCun, L. Bottou, Y. Bengio and P. Haffner, Proc. IEEE 86, 2278 (1998).</li>


<li id="ref-17">K. Fukushima, Biol. Cybern. 36, 193 (1980)</li>


<li id="ref-18">G. E. Hinton, ‚ÄúTraining Products of Experts by Minimizing Contrastive Divergence,‚Äù Neural Computation, vol. 14, no. 8, pp. 1771‚Äì1800, Aug. 2002</li>


<li id="ref-19">P. Sharma and A. Singh, ‚ÄúEra of deep neural networks: A review,‚Äù 2017 8th International Conference on Computing, Communication and Networking Technologies (ICCCNT), Jul. 2017</li>


<li id="ref-20">B. K. Spears et al., ‚ÄúDeep learning: A guide for practitioners in the physical sciences,‚Äù Physics of Plasmas, vol. 25, no. 8, p. 080901, Aug. 2018</li>


<li id="ref-21">J. V. Stone, ‚ÄúJames V Stone - Books,‚Äù Sheffield.ac.uk, 2022</li>


<li id="ref-22">C. B. Anfinsen, ‚ÄúPrinciples that Govern the Folding of Protein Chains,‚Äù Science, vol. 181, no. 4096, pp. 223‚Äì230, Jul. 1973</li>


<li id="ref-23">P. B. Moore, W. A. Hendrickson, R. Henderson, and A. T. Brunger, ‚ÄúThe protein-folding problem: Not yet solved,‚Äù Science, vol. 375, no. 6580, pp. 507‚Äì507, Feb. 2022</li>


<li id="ref-24">R. Zwanzig, A. Szabo, and B. Bagchi, ‚ÄúLevinthal‚Äôs paradox.,‚Äù Proceedings of the National Academy of Sciences, vol. 89, no. 1, pp. 20‚Äì22, Jan. 1992</li>


<li id="ref-25">B. Honig, ‚ÄúProtein folding: from the levinthal paradox to structure prediction,‚Äù Journal of Molecular Biology, vol. 293, no. 2, pp. 283‚Äì293, Oct. 1999</li>


<li id="ref-26">J. T. Seffernick and S. Lindert, ‚ÄúHybrid methods for combined experimental and computational determination of protein structure,‚Äù The Journal of Chemical Physics, vol. 153, no. 24, Dec. 2020</li>


<li id="ref-27">‚ÄúRCSB PDB: Homepage,‚Äù Rcsb.org. https://www.rcsb.org/</li>


<li id="ref-28">Google DeepMind, ‚ÄúAbout Google DeepMind,‚Äù Google DeepMind. https://deepmind.google/about/</li>


<li id="ref-29">A. W. Senior et al., ‚ÄúImproved Protein Structure Prediction Using Potentials from Deep Learning,‚Äù Nature, vol. 577, no. 7792, pp. 706‚Äì710, Jan. 2020</li>


<li id="ref-30">J. Jumper et al., ‚ÄúHighly Accurate Protein Structure Prediction with Alphafold,‚Äù Nature, vol. 596, no. 7873, pp. 583‚Äì589, Jul. 2021</li>


<li id="ref-31">C. Metz, ‚ÄúLondon A.I. Lab Claims Breakthrough That Could Accelerate Drug Discovery,‚Äù The New York Times, Nov. 30, 2020</li>


<li id="ref-32">J. Abramson et al., ‚ÄúAccurate structure prediction of biomolecular interactions with AlphaFold 3,‚Äù Nature, vol. 630, no. 630, pp. 493‚Äì500, May 2024</li>


<li id="ref-33">AlphaFold, ‚ÄúAlphaFold Protein Structure Database,‚Äù alphafold.ebi.ac.uk, 2022. https://alphafold.ebi.ac.uk/</li>


<li id="ref-34">M. L. Hekkelman, I. de Vries, R. P. Joosten, and A. Perrakis, ‚ÄúAlphaFill: enriching AlphaFold models with ligands and cofactors,‚Äù Nature Methods, Nov. 2022</li>


<li id="ref-35">F. Ren et al., ‚ÄúAlphaFold accelerates artificial intelligence powered drug discovery: efficient discovery of a novel CDK20 small molecule inhibitor,‚Äù Chemical Science, vol. 14, no. 6, 2023</li>


<li id="ref-36">M. Struthers, R. P. Cheng, and B. Imperiali, ‚ÄúDesign of a Monomeric 23-Residue Polypeptide with Defined Tertiary Structure,‚Äù Science, vol. 271, no. 5247, pp. 342‚Äì345, Jan. 1996</li>


<li id="ref-37">B. I. Dahiyat and S. L. Mayo, ‚ÄúDe Novo Protein Design: Fully Automated Sequence Selection,‚Äù Science, vol. 278, no. 5335, pp. 82‚Äì87, Oct. 1997</li>


<li id="ref-38">B. Kuhlman, G. Dantas, G. C. Ireton, G. Varani, B. L. Stoddard, and D. Baker, ‚ÄúDesign of a Novel Globular Protein Fold with Atomic-Level Accuracy,‚Äù Science, vol. 302, no. 5649, pp. 1364‚Äì1368, Nov. 2003</li>


<li id="ref-39">Simons, K.T., Bonneau, R., Ruczinski, I. and Baker, D. "Ab initio protein structure prediction of CASP III targets using ROSETTA". Proteins, 37: 171-176, 1999</li>


<li id="ref-40">https://www.ipd.uw.edu/</li>


<li id="ref-41">‚ÄúSolve Puzzles for Science | Foldit,‚Äù fold.it. https://fold.it/</li>


<li id="ref-42">Horowitz, S., Koepnick, B., Martin, R. et al. Determining crystal structures through crowdsourcing and coursework. Nat Commun 7, 12549 (2016)</li>


<li id="ref-43">H. Jo and D.-H. Park, ‚ÄúEffects of ChatGPT‚Äôs AI capabilities and human-like traits on spreading information in work environments,‚Äù Scientific Reports, vol. 14, no. 1, p. 7806, Apr. 2024</li>


<li id="ref-44">S.-J. Chen et al., ‚ÄúProtein folds vs. protein folding: Differing questions, different challenges,‚Äù Proceedings of the National Academy of Sciences, vol. 120, no. 1, Dec. 2022</li>


<li id="ref-45">V. Agarwal and A. C. McShan, ‚ÄúThe power and pitfalls of AlphaFold2 for structure prediction beyond rigid globular proteins,‚Äù Nature Chemical Biology, Jun. 2024</li>


<li id="ref-46">G. Conroy and S. Mallapaty, ‚ÄúHow China created AI model DeepSeek and shocked the world,‚Äù Nature.com, Jan. 2025</li>


<li id="ref-47">S. S. Haykin, Neural networks and learning machines. New York: Prentice Hall/Pearson, 2009</li>


<li id="ref-48">Popular information. NobelPrize.org. Nobel Prize Outreach 2025. Sun. 23 Feb 2025. https://www.nobelprize.org/prizes/physics/2024/popular-information</li>


</ol>
</div>


</main>

 <footer>
	<span>¬©Ô∏è 2025 InScight </span>
	<span>|</span>
	<span>üì§ Hosted by <a href="https://www.iiserkol.ac.in/web/en/" target="_blank">IISER Kolkata</a></span>
	<span>|</span>
	<span><a class="nf social-links nf-fa-instagram" href="https://www.instagram.com/inscight_iiserkolkata/">Û∞ãæ</a><a class="nf social-links nf-fa-twitter_square" href="https://x.com/InScightIISERK">Ó£â</a></span>
</footer> 

</body>
</html>

